bindings/flow/fdb_flow.actor.cpp

```diff
@@ -98,8 +98,83 @@ void fdb_flow_test() {
 }
 
 namespace FDB {
+class DatabaseImpl : public Database, NonCopyable {
+public:
+	virtual ~DatabaseImpl() { fdb_database_destroy(db); }
```

Comment:
Why is this destructor `virtual`?

Reply:
All `ReferenceCounted` objects have to have virtual destructors to get destructed from `Reference` class.

Or your question is why this has `virtual` keyword instead of `override`? If so, no specific reason, just thought it would be more obvious for destructor.